docker-compose.yml

```diff
@@ -1,7 +1,9 @@
 version: "2.1"
 services:
   db:
-    image: mysql:5.6
+    # Oracle-packaged version includes a `linux/arm64/v8` version, needed for
```

Comment:
This is necessary because the `JSONField` Django model field is only supported in MySQL version greater than or equal to `5.7`. This makes it impossible to run migrations because the models in `xblock-lti-consumer` use `JSONField`. Because we have migrated to version `8.0`, I've updated the base MySQL image to `8.0`. This particular image is copied from [devstack](https://github.com/openedx/devstack/blob/fa758e4e51aee3df8d69af975e26154c4cd70bab/docker-compose.yml#L229).